crates/oxc_semantic/src/builder.rs

```diff
@@ -31,6 +31,16 @@ use crate::{
     Semantic,
 };
 
+macro_rules! cfg {
```

Comment:
Sure, I was thinking we can use std::cfg directly if needed, Named it `cfg` since it is a repeating pattern all over this file and it looks like the comments. I'll change it to `control_flow`.

Reply:
Personally I find the use of "cfg" as abbreviation for "control flow graph" confusing in general. There's quite a strong convention in Rust that "cfg" is short for "config" e.g. `#[cfg(...)]`.

When I first started looking at semantic's code, I kept asking myself "what's this 'config' all about?". It didn't take me too long to figure it out, but I think it's useful to remember what your first encounter with the codebase felt like, to put yourself in the position of a new contributor (who of course we want to welcome). The principle of minimum surprise and all that...

That's a broader point, and not such a big deal. But on this specific one, personally I think overriding part of the standard prelude is a step too far! If you would be willing to change it, I think it'd be ideal.

Reply:
I've already changed it in a subsequent PR.

Reply:
Ha! I am too slow again! Thanks for doing that.

Reply:
No worries mate! Winds are blowing fast, "Jibe Ho" ⚓